components/tidb_query/src/expr_util/collation/utf8mb4.rs

```diff
@@ -0,0 +1,613 @@
+    [0x00A1, 0x00A1, 0x00A1],
+    [0x00A2, 0x00A2, 0x00A2],
+    [0x00A3, 0x00A3, 0x00A3],
+    [0x00A4, 0x00A4, 0x00A4],
+    [0x00A5, 0x00A5, 0x00A5],
+    [0x00A6, 0x00A6, 0x00A6],
+    [0x00A7, 0x00A7, 0x00A7],
+    [0x00A8, 0x00A8, 0x00A8],
+    [0x00A9, 0x00A9, 0x00A9],
+    [0x00AA, 0x00AA, 0x00AA],
+    [0x00AB, 0x00AB, 0x00AB],
+    [0x00AC, 0x00AC, 0x00AC],
+    [0x00AD, 0x00AD, 0x00AD],
+    [0x00AE, 0x00AE, 0x00AE],
+    [0x00AF, 0x00AF, 0x00AF],
+    [0x00B0, 0x00B0, 0x00B0],
+    [0x00B1, 0x00B1, 0x00B1],
+    [0x00B2, 0x00B2, 0x00B2],
+    [0x00B3, 0x00B3, 0x00B3],
+    [0x00B4, 0x00B4, 0x00B4],
+    [0x039C, 0x00B5, 0x039C],
+    [0x00B6, 0x00B6, 0x00B6],
+    [0x00B7, 0x00B7, 0x00B7],
+    [0x00B8, 0x00B8, 0x00B8],
+    [0x00B9, 0x00B9, 0x00B9],
+    [0x00BA, 0x00BA, 0x00BA],
+    [0x00BB, 0x00BB, 0x00BB],
+    [0x00BC, 0x00BC, 0x00BC],
+    [0x00BD, 0x00BD, 0x00BD],
+    [0x00BE, 0x00BE, 0x00BE],
+    [0x00BF, 0x00BF, 0x00BF],
+    [0x00C0, 0x00E0, 0x0041],
+    [0x00C1, 0x00E1, 0x0041],
+    [0x00C2, 0x00E2, 0x0041],
+    [0x00C3, 0x00E3, 0x0041],
+    [0x00C4, 0x00E4, 0x0041],
+    [0x00C5, 0x00E5, 0x0041],
+    [0x00C6, 0x00E6, 0x00C6],
+    [0x00C7, 0x00E7, 0x0043],
+    [0x00C8, 0x00E8, 0x0045],
+    [0x00C9, 0x00E9, 0x0045],
+    [0x00CA, 0x00EA, 0x0045],
+    [0x00CB, 0x00EB, 0x0045],
+    [0x00CC, 0x00EC, 0x0049],
+    [0x00CD, 0x00ED, 0x0049],
+    [0x00CE, 0x00EE, 0x0049],
+    [0x00CF, 0x00EF, 0x0049],
+    [0x00D0, 0x00F0, 0x00D0],
+    [0x00D1, 0x00F1, 0x004E],
+    [0x00D2, 0x00F2, 0x004F],
+    [0x00D3, 0x00F3, 0x004F],
+    [0x00D4, 0x00F4, 0x004F],
+    [0x00D5, 0x00F5, 0x004F],
+    [0x00D6, 0x00F6, 0x004F],
+    [0x00D7, 0x00D7, 0x00D7],
+    [0x00D8, 0x00F8, 0x00D8],
+    [0x00D9, 0x00F9, 0x0055],
+    [0x00DA, 0x00FA, 0x0055],
+    [0x00DB, 0x00FB, 0x0055],
+    [0x00DC, 0x00FC, 0x0055],
+    [0x00DD, 0x00FD, 0x0059],
+    [0x00DE, 0x00FE, 0x00DE],
+    [0x00DF, 0x00DF, 0x0053],
+    [0x00C0, 0x00E0, 0x0041],
+    [0x00C1, 0x00E1, 0x0041],
+    [0x00C2, 0x00E2, 0x0041],
+    [0x00C3, 0x00E3, 0x0041],
+    [0x00C4, 0x00E4, 0x0041],
+    [0x00C5, 0x00E5, 0x0041],
+    [0x00C6, 0x00E6, 0x00C6],
+    [0x00C7, 0x00E7, 0x0043],
+    [0x00C8, 0x00E8, 0x0045],
+    [0x00C9, 0x00E9, 0x0045],
+    [0x00CA, 0x00EA, 0x0045],
+    [0x00CB, 0x00EB, 0x0045],
+    [0x00CC, 0x00EC, 0x0049],
+    [0x00CD, 0x00ED, 0x0049],
+    [0x00CE, 0x00EE, 0x0049],
+    [0x00CF, 0x00EF, 0x0049],
+    [0x00D0, 0x00F0, 0x00D0],
+    [0x00D1, 0x00F1, 0x004E],
+    [0x00D2, 0x00F2, 0x004F],
+    [0x00D3, 0x00F3, 0x004F],
+    [0x00D4, 0x00F4, 0x004F],
+    [0x00D5, 0x00F5, 0x004F],
+    [0x00D6, 0x00F6, 0x004F],
+    [0x00F7, 0x00F7, 0x00F7],
+    [0x00D8, 0x00F8, 0x00D8],
+    [0x00D9, 0x00F9, 0x0055],
+    [0x00DA, 0x00FA, 0x0055],
+    [0x00DB, 0x00FB, 0x0055],
+    [0x00DC, 0x00FC, 0x0055],
+    [0x00DD, 0x00FD, 0x0059],
+    [0x00DE, 0x00FE, 0x00DE],
+    [0x0178, 0x00FF, 0x0059],
+];
+
+const GENERAL_CI_PLANE_01: [[u16; 3]; 256] = [
```

Comment:
In MySQL, there are many planes general_ci used, please add it.

Reply:
TiKV will follow TiDB's implementation. Meanwhile TiDB has not implemented, this is a sample interface in order to not blocking other issues.

Reply:
ok